java-client/session/src/main/java/io/deephaven/client/impl/BatchTableRequestBuilder.java

```diff
@@ -818,8 +825,18 @@ public Condition visit(FilterComparison comparison) {
 
         @Override
         public Condition visit(FilterIn in) {
-            // TODO(deephaven-core#3609): Update gRPC expression / filter / literal structures
```

Comment:
Lots more cases still UOE. Do we expect users to hit them soon?

Reply:
Probably. Would like to consider https://github.com/deephaven/deephaven-core/issues/3609 sooner rather than later.